sdk/storage/Azure.Storage.Files.Shares/tests/ShareClientTests.cs

```diff
@@ -1314,12 +1335,12 @@ public async Task SetPropertiesAsync_Error()
             // Arrange
             var shareName = GetNewShareName();
             ShareServiceClient service = GetServiceClient_SharedKey();
+            ShareClientOptions clientOptions = new ShareClientOptions(ShareClientOptions.ServiceVersion.V2020_02_10);
```

Comment:
we shouldn't be hardcoding SV like this.

Reply:
woops left this in there. I was trying to test each service version with this line. thanks for the catch